src/meta/client/MetaClient.cpp

```diff
@@ -130,6 +130,7 @@ void MetaClient::heartBeatThreadFunc() {
             localLastUpdateTime_ = metadLastUpdateTime_;
         }
     }
+    uploadSession();
```

Comment:
why not do it in `reclaimExpiredSessions`? and `pushSessionToCache` can delete

Reply:
The way you speak at the moment is feasible.
Currently, metad is updated using graphd session information.
If we support` kill session`, it needs metad information to update the graphd.